index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <title>Product landing page</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <header id="header">
+        <img src="img/company-logo.jpg" alt="logo" id="header-img">
+      <nav id="nav-bar">
+        <a class="nav-link" href="#features-anchor">Features</a>
+        <a class="nav-link" href="#video-anchor">Demo</a>
+        <a class="nav-link" href="#form">Plans</a>
+      </nav>
+    </header>
+    <form action="https://www.freecodecamp.com/email-submit" method="get"
+    id="form">
+    <label id="email-label">
+    <input name="email" type="email" placeholder="Enter your email" required>
```

Comment:
Consider associating the `<label>` with the `<input>` element, to do this you need to give the `<input>` an `id` attribute. The `<label>` then needs a `for` attribute whose value is the same as the input's `id`.

* Example 1:
  ```html
  <label for="email">Email address</label> <br>
  <input type="email" name="email" id="email">```


* Example 2: 
Alternatively, you can nest the <input> directly inside the <label> like this:

```html
  <label for="email">Email address <br>
       <input type="email" name="email" id="email">
  </label>
``` 

>Associating a `<label>`  with an `<input>` element offers some major advantages:
>The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
>You can click the associated label to focus/activate the input, as well as the input itself. This increased hit area provides an advantage to anyone trying to activate the input, including those using a touch-screen device.

You can check for more [info on this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label) 👍⭐